docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -857,11 +857,17 @@ include::ingest-node-common-processor.asciidoc[]
 
 [source,js]
 --------------------------------------------------
+PUT _ingest/pipeline/studentindex
 {
-  "convert": {
-    "field" : "url.port",
-    "type": "integer"
-  }
+  "description": "for student index converting from string id field to integer one ",
```

Comment:
Again, this raises the question where or what the student index is. I would keep it more general. What do you think is missing or ambigouous from the previous example?

Reply:
Agree, how about `describe pipeline` for more general purpose?

Reply:
Maybe you could decribe what the processor does. In this case it "converts the content of the `id` field to an integer". By callind the field e.g. "id_string" you could also make it clearer that the original field is suppoesd to not be an integer but a string.

Reply:
Agree w/ describe more clear description like `converts the content of the id field to an integer`. But for using "id_string" maybe not a good idea because it *was* string, but it will be changed to integer after ingestion. So, it can make more confusion.